### PR TITLE
Add static webp images

### PR DESCRIPTION
**Pull request by VladislavSokov**

### CODE REVIEW

**Comment by ka8725** [comment]
@VladislavSokov please explain in detail how that solution is supposed to work. I'm especially curious about what we do when add a new post with new images.

**Comment by VladislavSokov** [comment]
> @VladislavSokov please explain in detail how that solution is supposed to work. I'm especially curious about what we do when add a new post with new images.

When creating a new showcase, we need to add the original image and the image in webp format to the showcases folder.

File: src/showcases.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export interface ShowcasePreview {
 export interface ShowcaseBody {
   bannerTopTitle: ReactNode;
   bannerTopImageSrc: string;
+  bannerTopImageWebpSrc: string;
```
Comment (ka8725) [inline src/showcases.tsx]: Can it be made **optional**?

File: src/app/showcases/[slug]/page.tsx [hunk 2]
```diff
@@ -67,14 +67,19 @@ export default function ShowcasePage() {
...
-            <Image
-              src={`${showcase.body.bannerTopImageSrc}`}
-              priority
-              alt="Banner Image"
-              quality={100}
-              width={687}
-              height={377}
-            />
+            <picture>
+              { showcase.body.bannerTopImageWebpSrc !== '' &&
```
Comment (ka8725) [inline src/app/showcases/[slug]/page.tsx]: It it's optional there should no be hack with empty string

**Review by ka8725 — CHANGES_REQUESTED** [review]
The webp argument should be **optional**. Optional means no value provided for it is possible

**Pull request merged**